code/modules/mob/living/carbon/human/species_types/felinid.dm

```diff
@@ -14,6 +13,8 @@
 
 	mutantears = /obj/item/organ/ears/cat
 	mutanttail = /obj/item/organ/tail/cat
+	mutanttongue = /obj/item/organ/tongue/cat
```

Comment:
Who approved that?

Reply:
I have approval for a non-specific felinid change PR from TheFakeElon:
![image](https://user-images.githubusercontent.com/7960287/116238027-2cc97a80-a761-11eb-89df-355d398add7c.png)